crypto/keys/pem_signer_factory.go

```diff
@@ -21,17 +21,32 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"fmt"
+	"sync"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 	"github.com/google/trillian"
 	"github.com/google/trillian/crypto/keyspb"
 )
 
+var (
+	pkcs11Module string
```

Comment:
Why would the interface need such a method? Surely you'd create a PEMSignerFactory, call the method to set the module, then add it to the registry.